docker/hive/Dockerfile

```diff
@@ -11,6 +11,7 @@ FROM debian:bookworm as apibara-build
 RUN apt-get update && apt-get install -y patchelf && rm -rf /var/lib/apt/lists/*
 
 # Apibara Starknet and Sink are built with Nix, so we need to extract the binaries from the Nix store
+# ⚠️ When modifying the tag of Apibara, make sure to change the Github CI workflow, and Indexer dockerfile as well
```

Comment:
Also update the README file no? I think we have also the tags there as an instruction to build a local docker image of the RPC.